### PR TITLE
Add BUILD_REQUIRES to Makefile.PL

### PR DESCRIPTION
**Pull request by atoomic**
some tests require Devel::NYTProf

### CODE REVIEW

File: Makefile.PL [hunk 1]
```diff
@@ -51,6 +51,10 @@ WriteMakefile
    DISTNAME     => 'Filter',
    NAME         => 'Filter::Util::Call',
    VERSION_FROM => 'Call/Call.pm',
+   BUILD_REQUIRES => {
```
Comment (rurban) [inline Makefile.PL]: should be TEST_REQUIRES

Comment (atoomic) [reply Makefile.PL]: this is changed to TEST_REQUIRES

**Comment by atoomic** [comment]
@rurban just realized that the PR was updated with your requested changes
would appreciate a second review

**Comment by atoomic** [comment]
indeed I cannot find evidence that NYTProf is used... let's close this PR

**Comment by rurban** [comment]
Maybe it was some leftovers of nyt runs. But we don't care for these.

**Pull request closed without merging**